ios/app/brave_core_main.h

```diff
@@ -62,6 +63,8 @@ OBJC_EXPORT
 
 @property(nonatomic, readonly) BravePasswordAPI* passwordAPI;
 
+@property(nonatomic, readonly) BraveFavIconLoader* favIconLoader;
```

Comment:
I know its silly but the name's throughout the PR should be `Favicon`, not `FavIcon`